e2e/suites/metadataprops.test.ts

```diff
@@ -0,0 +1,166 @@
+/* eslint-disable max-statements */
```

Comment:
Nit: I think this could be avoided by wrapping the longer `its` into a `describe`, with `its` that cover some properties, like: "it should fill all the dates", "it should fill all the selects", and then "it should have all the texts filled correctly", etc.

Reply:
In general, I have asked Konz about the eslint, and he said that in tests it's accepted to go over the line limit. 

Specifically for this case, I think it's nice to have them together, in the original order.